src/handleAction.js

```diff
@@ -5,9 +5,13 @@ function isFunction(val) {
 }
 
 export default function handleAction(type, reducers) {
+  const typeValue = isFunction(type)
```

Comment:
We can't always convert `type` to string type because it could be a `Symbol`. There is a test for that https://github.com/acdlite/redux-actions/blob/master/src/__tests__/handleActions-test.js#L25 . By the way `toString()` has a slightly better performance https://jsperf.com/string-vs-tostring2